src/components/AddItem.js

```diff
@@ -1,16 +1,19 @@
 import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { db } from '../lib/firebase';
+import Error from './Error';
 
 const initialFormState = {
   itemName: '',
+  cleanedUpItemName: '',
   timeFrame: 7,
   lastPurchased: null,
 };
 
 const AddItem = ({ token }) => {
   const history = useHistory();
   const [formData, setFormData] = useState(initialFormState);
+  const [hidden, setHidden] = useState(false);
```

Comment:
Reading through this code as someone who is unfamiliar, it's not quite clear to me what `hidden` is going to be used for. Maybe it hides the form, or parts of it? Maybe something else?

I would recommend a variable name here that makes it super clear, such as `shouldShowError` or something like that. I always like to prefix my boolean values using `is` or `should` or something to indicate that it's a boolean flag.

Reply:
Oh hehe looks like @laurenmbeatty had the same suggestion here! https://github.com/the-collab-lab/tcl-17-smart-shopping-list/pull/23/files#r561408192